src/runners/global-search-runner.js

```diff
@@ -4,12 +4,28 @@ import GlobalSearch from '../../components/global_search/global-search.react';
 
 // import component
 
-ReactDOM.render(
-	<GlobalSearch desktop={true}/>,
-	document.getElementById('global-search-desktop-component'),
-);
-
-ReactDOM.render(
-	<GlobalSearch desktop={false}/>,
-	document.getElementById('global-search-mobile-component'),
-);
\ No newline at end of file
+
+function renderIfExists(id, desktop, defaultFormDataIndex) {
+
+	if(document.getElementById(id)) {
+		ReactDOM.render(
+			<GlobalSearch desktop={desktop} defaultFormDataIndex={defaultFormDataIndex}
+			/>,
+			document.getElementById(id)
+		)
+	}
+
```

Comment:
I wonder if we should remove these two additional line breaks - just a minor formatting thing.